### DISCUSSION TITLE
Error Loading Spacy pickle model to gpu

### DISCUSSION THREAD

**Discussion by rajeshmore1** [explosion/spaCy#13277] [Help: Coding & Implementations] [answered]
I am trying to load spacy pickled model using spacy.load("path/model.pkl") as I wanted to use gpu.

spacy.prefer_gpu()
spacy.load("path/model.pkl")

Above code giving me error.
 OSError: [E050] Can't find model '<spacy.lang.en.English object at 0x7e8e7d13ef20>'. It doesn't seem to be a Python package or a valid path to a data directory.
 
The data type of the model is spacy.lang.en.English, despite this I am getting error. Could you please suggest - how to use gpu to load the model to gpu and how to use it efficiently for inference?

## Your Environment

* Operating System: Windows
* Python Version Used: 3.9
* spaCy Version Used: spacy_transformers==1.3.3, spacy==3.6.1
* Environment Information:

**Comment by svlandeg** [comment]
Do you get the same error without the `spacy.prefer_gpu()` command?

**Comment by rajeshmore1** [comment, marked as answer]
Hi Team, Thank you for the suggestion. The following approach worked in my case.

I had model in .pkl format. I converted the model into spacy compatible format by using to_disk function. ,( .pkl file converted into multiple folder structure which was spacy gpu compatible) Aftar that I followed the same procedure that you recommended. Thank you.
